rmgpy/solver/base.pyx

```diff
@@ -746,20 +752,28 @@ cdef class ReactionSystem(DASx):
                             conversion = 1 - (y_core_species[index] / y0[index])
 
                     if invalid_objects == []:
-                        #species flux criterion
-                        if len(edge_species_rate_ratios) > 0:
-                            ind = np.argmax(edge_species_rate_ratios)
-                            obj = edge_species[ind]
-                            logging.info('At time {0:10.4e} s, species {1} at rate ratio {2} was added to model core '
-                                         'in model resurrection process'.format(self.t, obj,edge_species_rates[ind]))
-                            invalid_objects.append(obj)
-
-                        if total_div_accum_nums and len(total_div_accum_nums) > 0:  #if dynamics data available
-                            ind = np.argmax(total_div_accum_nums)
-                            obj = edge_reactions[ind]
-                            logging.info('At time {0:10.4e} s, Reaction {1} at dynamics number {2} was added to model core '
-                                         'in model resurrection process'.format(self.t, obj,total_div_accum_nums[ind]))
-                            invalid_objects.append(obj)
+                        if self.retry < self.max_retries:
+                            #species flux criterion
+                            if len(edge_species_rate_ratios) > 0:
+                                ind = np.argmax(edge_species_rate_ratios)
+                                obj = edge_species[ind]
+                                self.retry += 1
+                                logging.info('At time {0:10.4e} s, species {1} at rate ratio {2} was added to model core '
+                                            'in model resurrection process. Retry # {3} of {4}'.format(self.t, obj,edge_species_rates[ind], self.retry, self.max_retries))
+                                invalid_objects.append(obj)
+
+
+                            if total_div_accum_nums and len(total_div_accum_nums) > 0:  #if dynamics data available
+                                ind = np.argmax(total_div_accum_nums)
+                                obj = edge_reactions[ind]
+                                self.retry += 1
+                                logging.info('At time {0:10.4e} s, Reaction {1} at dynamics number {2} was added to model core '
+                                            'in model resurrection process. Retry # {3} of {4}'.format(self.t, obj,total_div_accum_nums[ind],  self.retry, self.max_retries))
+                                invalid_objects.append(obj)
+                                
+                        else:
+                            raise RuntimeError ('too many retries, ending simulation')
```

Comment:
I think RMG users might not know what `too many retries` means should they encounter it.  Perhaps we can make it a littler more descriptive:
`Reached the maximum permitted number of solver errors (self.max_retries), ending simulation`.

Reply:
Agreed. I changed it to the say something similar, as shown in the attached traceback
[resurrect_retry_traceback.txt](https://github.com/ReactionMechanismGenerator/RMG-Py/files/7687414/resurrect_retry_traceback.txt)
.